app/models/recruitment_cycle.rb

```diff
@@ -28,6 +28,10 @@ def self.next_recruitment_cycle
     all.order(:year).second
   end
 
+  def next
+    RecruitmentCycle.find_by(year: year.to_i + 1)
+  end
```

Comment:
Even though this is the right implementation, I feel like this muddies the waters a bit ... wrt `RecruitmentCycle#next_recruitment_cycle`. I know this is a pretty big PR already, but is there anything simple we can do to not have this split-view of what qualifies as the "next" cycle?

Reply:
Would it help if we changed:

```ruby
  def self.next_recruitment_cycle
    current_recruitment_cycle.next
  end
```

Reply:
Ya, I thought that would work too. 👍

Reply:
Right-o, fixed in 4d25757